### DISCUSSION TITLE
Checking duplicates return zero results

### DISCUSSION THREAD

**Discussion by peia86** [TheLastGimbus/GooglePhotosTakeoutHelper#293] [I NEED HELP] [answered]
Hi there,

first of all thanks for your excellent work.
Anyway, in may case is not working...
Everything is good untill the "check  of duplicate"... the program is still for few minutes, then it close without any message. I tried different method/folders but it is always the same.
Few info: I have a folder Takeout of around 70gb of files in multiple folders (albums), folder is in Synology NAS and are accessible throug SMB, I am using Windows 11. Let me know if there is a way to show you a log or other info.

thanks

**Comment by ikwyl6** [comment]
Hi - I would like to add my general comments here…
First - please change the title to something like ‘checking duplicates returns zero results’. Having something like ‘running but not working’ is the most vague title ever. You need to be short and decisive. This also helps future users who’ve had the same or similar issues to find this GitHub issue extremely easy when searching for it. 
Second Regarding logs - have you looked at the script to see if there is an option to be verbose or to print a log to a file? Just please do a little homework and look at the program first before just asking some general questions. This also helps you get faster answers as well if you provide more info and show you’ve done a few steps and a little homework before just saying it’s ’running but not working’…

**Reply by peia86** [reply]
Hi thank you for your suggestions, I changed the title.
I made homework but can't find a way to do a log, I am totally zero in scripting my help request it is also to know if there is a way to have a log. Thank you for your support.

**Comment by TheLastGimbus** [comment, marked as answer]
> first of all thanks for your excellent work.
> Anyway, in may case is not working

Then what are you thanking for :joy: - just kidding, thanks

Issue exactly like this popped a few times here, and i couldn't find any reason for those cases - but for yours, i think there might be

You see gpth needs to have access to read, analyze, and move a lot of files. In your case, 70gb of them. And, especially the dupilacte-finding process needs to read both files and calculate their sha256 checksums to compare if they are **exactly the same** file. So, your computer will have to, at the end, download... let's say, 20gb of files over samba (and, potentially, hold them in ram or something) to calulate them. That's infinietley slower than if you just had them at usb drive

Second - i have no idea if Dart's filesystem access plays nice with netwrok drives etc etc

Solution: run everything on your PC, or, your Synology: look at #157 for knowlage :brain:

**Reply by peia86** [reply]
Thank you for your support, that is a smart solution that I tought to try. 
But, before to try your suggestion I tried to re-unzip the takeout in a new folder (maybe in my first attempt I forgot to unzip one or more package) and also I changed the destination folder with an empty one (my first attempt was in a folder with other files).  Now GPTH is working! I am just at half of the data processed, but it is going smoothly. I do not know if because the change I made or for other reason.

Thank you for your work and support! I will give you a feedback (and tips ;) at the end of the process.

**Reply by TheLastGimbus** [reply]
Try to do it locally anyway - it will be faster and smb *mayyy* break some stuff 🤷‍♂️🤷‍♂️🤷‍♂️ idk

**Reply by peia86** [reply]
Well job done!.. 
Just an issue about albums... the script created shortcut for pictures in folders but the link point to a wrong directory, for example: the picture in Z:/photos/nameofthealbum/photos01.jpg now is a shortcut pointing to C:/photos/nameofthealbum/photos01.jpg so the link is corrupted. Let me know if I am missing something or if it is a bug.
Thank you!

**Reply by TheLastGimbus** [reply]
Yeah it seems that albums are buggy for now - will take care of this in... Some time

Happy it works!
